packages/rich-text/src/test/index.native.js

```diff
@@ -109,6 +109,20 @@ describe( '<RichText/>', () => {
 			expect( actualFontSize ).toBe( expectedFontSize );
 		} );
 
+		it( `should display rich text at the PROVIDED font size computed from the selected GLOBAL
+		\`__experimentalGlobalStylesBaseStyles.typography.fontSize\` CSS with decimal values.`, () => {
+			// Arrange
+			const expectedFontSize = 18;
+			mockGlobalSettings( { fontSize: '1.125rem' } );
```

Comment:
Should we test out a few different sizes here?

Reply:
Thanks for the suggestion @enejb! I added a few more =) https://github.com/WordPress/gutenberg/pull/37027/commits/7da2bf7ee20e1b9b875f383c1fe63f20d3ae8dfb